### PR TITLE
CSP-325-DotCore6Migration-apply

### PR DESCRIPTION
**Pull request by raviudari01**

### CODE REVIEW

File: src/SFA.DAS.ApplyService.Data/OrganisationRepository.cs [hunk 1]
```diff
@@ -111,18 +111,6 @@ public async Task UpdateOrganisation(Organisation organisation, Guid userId)
             }
         }
 
-        public async Task UpdateOrganisation(Guid organisationId, Guid userId)
```
Comment (MarkFCain) [inline src/SFA.DAS.ApplyService.Data/OrganisationRepository.cs:114]: I wouldn't make any code changes, can you put this back?

**Review by MarkFCain — CHANGES_REQUESTED** [review]
Can you reset the index name back to what it was, and put back in the method OrganisationRepository.UpdateOrganisation

File: src/SFA.DAS.ApplyService.Data/SFA.DAS.ApplyService.Data.csproj [hunk 2]
```diff
@@ -13,7 +13,7 @@
 
     <ItemGroup>
       <PackageReference Include="Dapper">
-        <Version>1.50.5</Version>
```
Comment (DevenKShah) [inline src/SFA.DAS.ApplyService.Data/SFA.DAS.ApplyService.Data.csproj:16]: Move version on the same line

**Review by MarkFCain — DISMISSED** [review]

File: src/SFA.DAS.ApplyService.Web/Views/Home/Error.cshtml [hunk 3]
```diff
@@ -10,32 +8,9 @@
 <main class="govuk-main-wrapper govuk-main-wrapper--l" id="main-content" role="main">
     <div class="govuk-grid-row">
         <div class="govuk-grid-column-two-thirds">
-            @if (HostingEnvironment.IsDevelopment())
```
Comment (DevenKShah) [inline src/SFA.DAS.ApplyService.Web/Views/Home/Error.cshtml:13]: We removed this with agreement that in the startup we will configure to go to this page only if it is on hosted environment, we still need to see `DeveloperExceptionPage` when debugging. I have not found any config changes in this regards?

**Review by DevenKShah — COMMENTED** [review]
Developer exception page needs to be enabled.

File: src/SFA.DAS.ApplyService.Web/Startup.cs [hunk 4]
```diff
@@ -36,20 +33,25 @@
...
 {
+    [ExcludeFromCodeCoverage]
     public class Startup
     {
         private readonly IConfiguration _configuration;
         private readonly ILogger<Startup> _logger;
-        private readonly IHostingEnvironment _hostingEnvironment;
-        private readonly IHostingEnvironment _env;
+        private readonly IWebHostEnvironment _hostingEnvironment;
+        private readonly IWebHostEnvironment _env;
```
Comment (DevenKShah) [inline src/SFA.DAS.ApplyService.Web/Startup.cs:49]: Do we need two variables for the same thing?

**Review by DevenKShah — DISMISSED** [review]

**Review by MarkFCain — DISMISSED** [review]

**Review by HussainsGitHub — APPROVED** [review]

**Review by DevenKShah — APPROVED** [review]

**Pull request merged**